MdePkg/Include/Library/PanicLib.h

```diff
@@ -0,0 +1,78 @@
+/** @file
+  Provides services to print panic messages to a debug output device.
+
+  Note that a reserved macro named MDEPKG_NDEBUG is brought from debuglib for the intention
+  of size reduction when compiler optimization is disabled. If MDEPKG_NDEBUG is
+  defined, then panic related macros wrapped by it are the NULL implementations.
+
+Copyright (c) 2023, Microsoft Corporation. All rights reserved.<BR>
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef __PANIC_LIB_H__
+#define __PANIC_LIB_H__

```

Comment:
```suggestion
#ifndef PANIC_LIB_H_
#define PANIC_LIB_H_

```

nit: This seems to be the more canonical style.

Reply:
And required by PatchCheck now.

Reply:
Thanks.  Updated.